polynote-kernel/src/test/resources/log4j.properties

```diff
@@ -0,0 +1,5 @@
+log4j.rootCategory=INFO, console
```

Comment:
During test runs it was complaining about log4j. Not sure if this did anything or not.